snooty/diagnostics.py

```diff
@@ -342,3 +342,14 @@ def __init__(
         end: Union[None, int, Tuple[int, int]] = None,
     ) -> None:
         super().__init__("Monospace text uses two backticks (``)", start, end)
+
+
+class IncorrectLinkSyntax(Diagnostic):
+    severity = Diagnostic.Level.error
+
+    def __init__(
+        self,
+        start: Union[int, Tuple[int, int]],
+        end: Union[None, int, Tuple[int, int]] = None,
+    ) -> None:
+        super().__init__("Malformed external link", start, end)
```

Comment:
It may be useful to provide an example of how external links should look here.

Reply:
That's actually done by the upcoming Did You Mean feature! Do you think it's important to both describe the correct syntax as well as show a corrected version?